client/src/components/Photo.js

```diff
@@ -0,0 +1,77 @@
+import React, {useEffect, useState} from "react";
+import Axios from 'axios';
+import {Image} from 'cloudinary-react'
+
+
+
+function Photo({user}){
+const [imageSelected, setImageSelected] = useState("");
```

Comment:
Look good. Maybe you can indent the code inside the function Photo so that it look good